# CREATE A NEW GROUP
uint64 parent_id
string group_name
---
uint8 id
bool success